test/compiler.test.js

```diff
@@ -514,6 +514,37 @@ describe('compiler', function() {
       expect(instructions.files.boot).to.eql([appJs]);
     });
 
+    it('prefers coffeescript over json in `appRootDir/bootDir`', function() {
+      // add coffee-script to require.extensions
+      require('coffee-script/register');
```

Comment:
Please don't register coffee script in every test. Move these two lines to the top of the test file, where other `require` calls are made.